lib/dpl/provider/scalingo.rb

```diff
@@ -6,57 +6,28 @@
 module DPL
   class Provider
     class Scalingo < Provider
-
       def install_deploy_dependencies
-        unless context.shell "curl -OL https://cli-dl.scalingo.io/release/scalingo_latest_linux_amd64.tar.gz && tar -zxvf scalingo_latest_linux_amd64.tar.gz && mv scalingo_*_linux_amd64/scalingo . && rm scalingo_latest_linux_amd64.tar.gz && rm -r scalingo_*_linux_amd64"
+        unless context.shell 'curl -OL https://cli-dl.scalingo.io/release/scalingo_latest_linux_amd64.tar.gz && tar -zxvf scalingo_latest_linux_amd64.tar.gz && mv scalingo_*_linux_amd64/scalingo . && rm scalingo_latest_linux_amd64.tar.gz && rm -r scalingo_*_linux_amd64'
           error "Couldn't install Scalingo CLI."
         end
       end
 
       def initialize(context, options)
         super
+        raise Error, 'you must specify your app name with `--app`' if options[:app] && options[:app].empty?
+
         @options = options
-        @remote = options[:remote] || "scalingo"
-        @branch = options[:branch] || "master"
+        @remote = "git@scalingo.com:#{options[:app]}.git"
```

Comment:
This makes the `app` option mandatory. And existing deployments without it will fail.